assets/src/main/java/bisq/asset/coins/DarkPay.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.Base58BitcoinAddressValidator;
+import bisq.asset.Coin;
+import bisq.asset.NetworkParametersAdapter;
+
+public class DarkPay extends Coin {
+    public DarkPay() {
+        super("DarkPay", "D4RK", new Base58BitcoinAddressValidator(new DarkPayMainNetParams()), Network.MAINNET);
```

Comment:
`Network.MAINNET` is default so remove that param.

Reply:
Hello, thanks for info, done :
https://github.com/bisq-network/bisq/pull/2918/commits/0a6c4c7cad5d7b154fe6c012a7f980c48bbd61cc